implementations/py/README.md

```diff
@@ -13,19 +13,17 @@ Http plugin currently supports two different methods `GET` and `POST`. Similar t
 Below is sample invocation of the `GET` request with custom request headers and query parameters (`urlParams`).
```

Comment:
in the title of this file, it should be `@polywrap/http-plugin-py` instead of `@polywrap/http-plugin-js`

Reply:
Let's use the name of the python package, so it'd be `polywrap-http-plugin`